FordLabs/PeopleMover
# Role UI Changes

## Issue
Connects #427 

## What was done
- [x] Adjusted margins of trait warning

## How to test
1. Open my roles modal and check margins on trait warning at the bottom of the modal
